### ISSUE TITLE
Add Deobfuscation techniques.

### ISSUE THREAD

**Issue by Yet-Zio** [abhi-r3v0/Adhrit#22]
Most Android applications nowadays use various obfuscation techniques and libraries. Similarly, there are many deobfuscation libraries out there, include some deobfuscation techniques in your project to make it more popular. By the way, your project will be a gift to reverse engineers.

**Comment by PSNAppz** [comment]
@Yet-Zio Thank you for the idea. Actually it's  true. Are you interested in working on this?

**Comment by Yet-Zio** [comment]
Yes I'm interested in this project but I'm currently working on a personal project at this time, so I can help you find some deobfuscation libraries that are useful.

**Comment by PSNAppz** [comment]
Great! That will do it.

**Comment by PSNAppz** [comment]
@abhi-r3v0 Should we integrate [android-reverse-r](https://github.com/justingarrick/android-reverse-r) with Adhrit? It looks perfect.
